types/chart.js/index.d.ts

```diff
@@ -620,6 +621,7 @@ declare namespace Chart {
         hoverBackgroundColor?: ChartColor | ChartColor[] | Scriptable<ChartColor>;
         hoverBorderColor?: ChartColor | ChartColor[] | Scriptable<ChartColor>;
         hoverBorderWidth?: number | number[] | Scriptable<number>;
+        hoverRadius?: number;
```

Comment:
You didn't provided any link to the part of the documentation/code which explain your changes yet. Please review the PR template and complete it accordingly.

Reply:
I got changes from here
https://www.chartjs.org/docs/latest/charts/bubble.html

Reply:
According to the docs, this property is also Scriptable.